### DISCUSSION TITLE
Make Dual Talent Specialization more affordable and expand it to Triple (or even Quadruple) Talent Specialization

### DISCUSSION THREAD

**Discussion by Hasbakse** [chromiecraft/chromiecraft#5873] [Suggestions]
Dear Devs,

I strongly recommend implementing the following changes to the Dual Talent Specialization (DTS) system:

Make DTS more affordable:
- Allows lower (and higher) level players to explore multiple specializations at an earlier stage, thereby enhancing their gameplay experience and satisfaction;
- Faster RDF que times as players discover multiple roles;
- Generally improves overall group finding experience which results in a smoother gameplay experience.

Introducing a third and/or fourth talent slot
- Encourages players to explore multiple specs, including DPS, heal, and tank roles, resulting in improved RDF que times and increased player satisfaction.
- Enhances the PvP experience as more players are willing to engage in PvP content whilst their preferred PvE specialization(s) (particularly Glyphs) remain intact. 
- Eliminates the frustrating task of frequent respecs for PvP or PvE, including the costly requirement of changing Glyphs, thus allowing players to enjoy various aspects of the game without constant adjustments. 

I recommend initiating a poll on Discord to gauge the player's receptiveness towards these proposed changes to the DTS system and hopefully see these implemented in the future. I believe these changes will positively impact the CC community. 

Thank you for your attention and your continued work.

Demize

**Comment by Wosser** [comment]
Dual specc is one of the few things that take a meaningful amount of gold out of the game and therefore important to fight inflation. If you make dual specc more affordable it will make crafting materials less affordable in the longterm.

**Comment by balleny** [comment]
> * Allows lower (and higher) level players to explore multiple specializations at an earlier stage, thereby enhancing their gameplay experience and satisfaction;

respec are free for this purpose since the first bracket (level 19 bracket)

> * Generally improves overall group finding experience which results in a smoother gameplay experience.

group finding experience, maybe yes. at the same time, you will have more players that dont "usually" play that spec which could affect the group also negatively


> Introducing a third and/or fourth talent slot

I assume this aint possible without client side modifications. respecs are free hence, you can also keep one regular spec and use the dual talents for experiments.
Its also quite non-blizzlike - but CC stuff unfortunately diverts more and more into custom server instead of focusing on AZ basics

> * Enhances the PvP experience as more players are willing to engage in PvP content whilst their preferred PvE specialization(s) (particularly Glyphs) remain intact.

we are (for quite a while) in a dead spot pvp wise. people will invest more into that once we reach arena.

> * Eliminates the frustrating task of frequent respecs for PvP or PvE, including the costly requirement of changing Glyphs, thus allowing players to enjoy various aspects of the game without constant adjustments.

also, regarding the point of glyphs: this would also kind-of invalidate inscription as profession if you artificially reduce the demand for glyphs. right now you have the regular glyphs available from inscriptors, and wotlk glyphs available from gold or honor (which is also an incentive to play pvp)
or do you plan on giving out gold to inscriptors instead?

> I recommend initiating a poll on Discord to gauge the player's receptiveness towards these proposed changes to the DTS system and hopefully see these implemented in the future. I believe these changes will positively impact the CC community.

Chromiecraft implemented a system with reduced costs for dual talents which increased with each content phase to be appropriate (currently, we are at 900g instead of 1000g).

**Comment by Amrx2** [comment]
Hi @Hasbakse,

Both Wosser & balleny summed it up nicely above. Dualspec cost is there as a goldsink, which outside of flying and bags from Shattrath is the only real gold sink in game currently. Respeccing costs are free however so you can spec anything you like to trial.

Changing talent slots to a triple or even quad spec requires custom patches in order for it to function and would be a requirement for all, therefore it isn't something we will be implementing (it is important to us for those who want to play on a clean client can do so).

Thank you for the suggestion nevertheless, this isn't something we would be implementing however.

Thanks,
Construct

**Comment by Hasbakse** [comment]
Thanks everyone on the prompt responses. I appreciate the arguments presented above regarding my proposed system, even though our opinions on the matter differs. 

> respec are free for this purpose since the first bracket (level 19 bracket)

Certainly, but when you're leveling you usually don't do it in a healing or tanking spec (exceptions aside), thus you'd have to sit near a class trainer waiting for RDF que to pops up before you can play and/or level up (or heal/tank with dps talents which severely limits your ability to understand that spec)

> group finding experience, maybe yes. at the same time, you will have more players that dont "usually" play that spec which could affect the group also negatively

Which matters less at level 40 where dungeons are easier than lvl 70 where people expect (normal) dungeons to go smoothly. This would just delay the issue. Whilst making it available at a lower level means people are already gradually adjusting and understanding alternative specs. This is is a problem I will likely face very soon once my Paladin hits level 70 and I have to learn both Protection and Holy specs from relative scratch. 

> I assume this aint possible without client side modifications. respecs are free hence, you can also keep one regular spec and use the dual talents for experiments.
Its also quite non-blizzlike - but CC stuff unfortunately diverts more and more into custom server instead of focusing on AZ basics

I fully agree with these points. I am also not a fan of making it more and more non-blizzlike due to, for a lack of a better word, valid slippery slope arguments. 

> Chromiecraft implemented a system with reduced costs for dual talents which increased with each content phase to be appropriate (currently, we are at 900g instead of 1000g).

That sounds fair!

> Changing talent slots to a triple or even quad spec requires custom patches in order for it to function and would be a requirement for all, therefore it isn't something we will be implementing (it is important to us for those who want to play on a clean client can do so).

Thank you for replying, I fully understand and sympathize with your decision not to do so. 

Thank you again for your thoughtful responses, I have a better understanding of the potential downsides of my proposed system, although I would like DTS to be affordable, I understand why perhaps we shouldn't. 

See you in-game

Demize
